index.html

```diff
@@ -83,6 +83,204 @@ <h1>
+        <div class="flex">
+            <div class="work-card">
+              <img src="img/IMG 3.png" alt="Tonic" />
+              <div class="work-card1">
+                <h2>Multi-Post Stories</h2>
+                <div class="canopy">
+                  <span>CANOPY</span>
+                  <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+                <p class="daily">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul class="lang">
+                  <li class="lang-item">html</li>
+                  <li class="lang-item">css</li>
+                  <li class="lang-item1">javascript</li>
+                </ul>
+                <button class="project-button">See Project</button>
+              </div>
+            </div>
+          </div>
+
+           <!-- TONIC2 -->
+        
+        <div class="flex">
+            <div class="work-card">
+              <img src="img/IMG 2.png" alt="Tonic" />
+              <div class="work-card1">
+                <h2>Tonic</h2>
+                <div class="canopy">
+                  <span>CANOPY</span>
+                  <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+                <p class="daily">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul class="lang">
+                  <li class="lang-item">html</li>
+                  <li class="lang-item">css</li>
+                  <li class="lang-item1">javascript</li>
+                </ul>
+                <button class="project-button">See Project</button>
+              </div>
+            </div>
+          </div>
+
+           <!-- MULTI-POST STORIES2 -->
+        
+        <div class="flex">
+            <div class="work-card">
+              <img src="img/IMG 4.png" alt="Tonic" />
+              <div class="work-card1">
+                <h2>Multi-Post Stories</h2>
+                <div class="canopy">
+                  <span>CANOPY</span>
+                  <ul>
+                    <li>Back End Dev</li>
+                    <li>2015</li>
+                  </ul>
+                </div>
+                <p class="d-text">
+                  A daily selection of privately personalized reads; no accounts
+                  or sign-ups required.
+                </p>
+                <ul class="lang">
+                  <li class="lang-item">html</li>
+                  <li class="lang-item">css</li>
+                  <li class="lang-item1">javascript</li>
+                </ul>
+                <button class="project-button">See Project</button>
+              </div>
+            </div>
+        </div>
+
+
+        <div class="main-container">
```

Comment:
- [ ] Kindly wrap your `about me content` in a `<section>` tag for semantic reasons.